config/groups.yaml

```
@@ -612,3 +612,9 @@
     - "2575698"
     - "2624077"
   id: 190035
+  - name: "SSH - snowman "
+  members:
+    - "2551905"
```

This isn't valid YAML. It would need to be this:
```suggestion
- name: "SSH - snowman "
  members:
    - "2551905"
```